cdc-quickstart-kafka-connect/grafana-dashboard/metrics-dashboard.json

```diff
@@ -1,703 +1,841 @@
+              "mode": "none"
+            },
+            "thresholdsStyle": {
+              "mode": "off"
             }
           },
-          "overrides": []
-        },
-        "gridPos": {
-          "h": 8,
-          "w": 12,
-          "x": 0,
-          "y": 0
-        },
-        "id": 16,
-        "options": {
-          "legend": {
-            "calcs": [],
-            "displayMode": "list",
-            "placement": "bottom",
-            "showLegend": true
-          },
-          "tooltip": {
-            "mode": "single",
-            "sort": "none"
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
           }
         },
-        "targets": [
+        "overrides": [
           {
-            "datasource": {
-              "type": "prometheus",
-              "uid": "P1809F7CD0C75ACF3"
+            "matcher": {
+              "id": "byName",
+              "options": "{tablet=\"03b1ec737ffb4090a0358c4f86429be2\"}"
             },
-            "editorMode": "builder",
-            "expr": "debezium_metrics_totalnumberofeventsseen{context=\"streaming\"}",
-            "legendFormat": "__auto",
-            "range": true,
-            "refId": "A"
+            "properties": [
+              {
+                "id": "color",
+                "value": {
+                  "fixedColor": "#B877D9",
+                  "mode": "fixed"
+                }
+              }
+            ]
           }
-        ],
-        "title": "debezium_metrics_totalnumberofeventsseen time series",
-        "type": "timeseries"
+        ]
       },
-      {
-        "datasource": {
-          "type": "prometheus",
-          "uid": "P1809F7CD0C75ACF3"
+      "gridPos": {
+        "h": 10,
+        "w": 12,
+        "x": 0,
+        "y": 0
+      },
+      "id": 18,
+      "options": {
+        "legend": {
+          "calcs": [],
+          "displayMode": "list",
+          "placement": "bottom",
+          "showLegend": true
+        },
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "P1809F7CD0C75ACF3"
+          },
+          "editorMode": "builder",
+          "expr": "sum by(tablet) (debezium_metrics_totalnumberofeventsseen{context=\"streaming\"})",
+          "hide": false,
+          "legendFormat": "count for {{tablet}}",
+          "range": true,
+          "refId": "A"
         },
-        "fieldConfig": {
-          "defaults": {
-            "color": {
-              "mode": "palette-classic"
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "P1809F7CD0C75ACF3"
```

Comment:
I may not be understanding this correctly but if this `uid` doesn't remain the same every time, it will cause issues, right?

Reply:
The JSON is generated by Grafana when dashboard is first created.To create the same dashboard subsequently, we just need to provide this JSON to it. Everytime it loads the dashboards using JSON it needs a uid. This uid can either be provieded in the JSON, or it can be auto generated. Having it autogenerated causes a problem where metrics are not set in the dashboard on startup hence using this way of providing UID.